elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -1037,18 +1061,71 @@ public String getId(Object value) {
      * @return class with Entity annotation
      */
     public Class<?> lookupEntityClass(Class<?> objClass) {
-        for (Class<?> cls = objClass; cls != null; cls = cls.getSuperclass()) {
-            EntityBinding binding = entityBindings.getOrDefault(cls, EMPTY_BINDING);
-            if (binding != EMPTY_BINDING) {
-                return binding.entityClass;
+        Class<?> declaringClass = lookupAnnotationDeclarationClass(objClass, Entity.class);
+        if (declaringClass != null) {
+            return declaringClass;
+        }
+        throw new IllegalArgumentException("Unbound Entity " + objClass);
+    }
+
+    /**
+     * Follow for this class or super-class for Include annotation.
+     *
+     * @param objClass provided class
+     * @return class with Include annotation or
+     */
+    public Class<?> lookupIncludeClass(Class<?> objClass) {
+        Annotation first = getFirstAnnotation(objClass, Arrays.asList(Exclude.class, Include.class));
```

Comment:
I think we should update `getFirstAnnotation()` as well, for this scenario 
```java
@Exclude 
class Foo { }

@Include
class Bar extends Foo {}

class Baz extends Bar {}
```
I think `getFirstAnnotation(Baz.class, Exclude.class, Include.class)` would return `Exclude.class`?

Reply:
Added a test.  It does the right thing.